content/en/Integrations/IntegrationBuilder/findings-outlook-notifications.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Outlook email notifications"
+linkTitle: "Configure outlook email notifications"
+weight: 40
+---
+
+## Configure outlook email notifications for pending pentest findings
+
+In order to configure Outlook to send emails every time a new finding gets created, we need to do the following:
+1. Create a Cobalt connection.
+2. Create an Outlook connection.
+3. Find and select a recipe (2 options):
+  A. [Cobalt > Outlook] When pentest finding moved to Pending Fix, send email notification
+  B. [Cobalt > Outlook] When DAST finding is detected, send email notification
+4. Copy and customize the recipe
+  1. Configure which pentest should trigger notifications when findings get created.
+  2. Configure which email address to send the to.
+5. Start the recipe
+
+The following is a step-by-step guide to walk you through the configuration process.
+
+### 1. Create a Cobalt connection.
+
+Find out how to create a Cobalt Connector [here](integrations/integrationbuilder/#connection).
```

Comment:
These URLs need are broken. They need to be referenced explicitly as a full path. For example:
```suggestion
Find out how to create a Cobalt Connector [here](/integrations/integrationbuilder/#connection).
```